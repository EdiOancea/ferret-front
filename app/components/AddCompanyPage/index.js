import React from 'react';
import CreateOutlinedIcon from '@material-ui/icons/CreateOutlined';

import PaperFrame from 'components/PaperFrame';
import AddCompanyForm from 'containers/AddCompanyForm';

const AddCompanyPage = () => (
  <PaperFrame headerText="Add Company" icon={<CreateOutlinedIcon />}>
    <AddCompanyForm />
  </PaperFrame>
);

export default AddCompanyPage;
