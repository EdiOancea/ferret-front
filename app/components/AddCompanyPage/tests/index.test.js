import React from 'react';
import { shallow } from 'enzyme';

import AddCompanyPage from 'components/AddCompanyPage';

describe('<AddCompanyPage />', () => {
  it('should not regress', () => {
    const tree = shallow(
      <AddCompanyPage>
        <div> a </div>
      </AddCompanyPage>,
    );
    expect(tree.children.length).toBe(1);
    expect(tree).toMatchSnapshot();
  });
});
