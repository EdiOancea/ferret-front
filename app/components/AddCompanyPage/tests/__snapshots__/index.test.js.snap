// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddCompanyPage /> should not regress 1`] = `
<WithStyles(PaperFrame)
  headerText="Add Company"
  icon={<pure(CreateOutlinedIcon) />}
>
  <withSaga(withReducer(Connect(AddCompanyForm))) />
</WithStyles(PaperFrame)>
`;
